### PR TITLE
Additional sample image and updated README diagram

### PR DESCRIPTION
**Pull request by MikhailRyazanov**
The lin-BASEX artificial image is added to `tools.analytical.SampleImage` and used as the README example (see #351).

The “overview” diagram (in README and “Comparison of Abel Transform Methods”) is also updated according to @DanHickstein's suggestions (see #350).

(I've also ran `pycodestyle` on `examples/` and included most suggested formatting corrections — some of them were in fact done during previous changes, but I forgot to include them in corresponding PRs. There are also minor correction to `tools.vmi` docstrings, also lingering from previous PRs.)

### CODE REVIEW

**Review by stggh — APPROVED** [review]
A nice image. Good a piece of work in reverse engineering it.

**Review by DanHickstein — COMMENTED** [review]
Was doc/overview.pdf added intentionally? I thought that you were just using the svg version. (Fine to add it, just making sure it wasn't a mistake.)

File: doc/readme_link.rst [hunk 1]
```diff
@@ -29,8 +29,8 @@
 
     fig, axs = plt.subplots(1, 2, figsize=(6, 3))
 
-    axs[0].imshow(forward_abel, cmap='ocean_r')
-    axs[1].imshow(inverse_abel, cmap='ocean_r')
+    axs[0].imshow(forward_abel, clim=(0, None), cmap='ocean_r')
```
Comment (DanHickstein) [inline doc/readme_link.rst:32]: Oh, I didn't realize that you could pass `None` to the clim arg! I'm going to use this!! :)

**Review by DanHickstein — APPROVED** [review]
Awesome work Mikhail! Looks fantastic to me.

File: doc/readme_link.rst (hunk 1 above)
Comment (MikhailRyazanov) [reply doc/readme_link.rst:32]: It seems that using `None` for “default” is a general convention. Better check the documentation, though... :–)

**Comment by MikhailRyazanov** [comment]
> Was doc/overview.pdf added intentionally? I thought that you were just using the svg version. (Fine to add it, just making sure it wasn't a mistake.)

Yes, it must be added for building the PDF documentation. The problem is that pdfLaTeX doesn't understand SVG images, and SVG cannot be converted to PDF automatically (at least not easily on RTD). Thus I've converted it manually (the instructions are included inside the SVG file) and force-added the resulting PDF file to the repository.

> Looks fantastic to me.

Great, thank you!

**Comment by MikhailRyazanov** [comment]
A note for future ourselves:

GitHub caches external images, and the “Example of use” output image in README, which is taken from RTD “latest” build, might get stuck here at its previous version. I had to [purge the cache](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-anonymized-urls#removing-an-image-from-camos-cache) by executing
```
curl -X PURGE https://camo.githubusercontent.com/81e63f8922d2a3c8b1ddd18cd1eb5000b3a89fb5c585050d649813a2344a22d0/68747470733a2f2f70796162656c2e72656164746865646f63732e696f2f656e2f6c61746573742f5f696d616765732f726561646d655f6c696e6b2d312e737667
```
(or whatever URL it will have) to make GitHub display the new version as well.

**Pull request merged**